spec/ParseRole.spec.js

```diff
@@ -49,7 +49,7 @@ describe('Parse Role testing', () => {
     }).then((x) => {
       x.set('foo', 'baz');
       // This should fail:
-      return x.save();
+      return x.save({},{sessionToken: ""});
```

Comment:
I change this because, by default the sessionToken is passed. The result of this never fail.